### ISSUE TITLE
SyntaxError: Unexpected token export

### ISSUE THREAD

**Issue by geraldo** [geostyler/geostyler-cli#64]
I run geostyler command on your testdata with the command `geostyler-cli -s sld -t qgis -o output.qml point_simplepoint.sld` and get the following error:

```
export * from './style';
^^^^^^

SyntaxError: Unexpected token export
```

I installed geostyler with _npm 7.16.0_ using sudo npm install -g geostyler-cli and no error shows up, but some warnings:

```
npm WARN deprecated @types/loglevel@1.6.3: This is a stub types definition. loglevel provides its own type definitions, so you do not need this installed.
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated url-search-params@1.1.0: now available as @ungap/url-search-params
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

Any hint?

**Comment by KaiVolland** [comment]
I'll have a look at it.

**Comment by KaiVolland** [comment]
We released geostyler-style in v 4.0.3 which includes [this fix](https://github.com/geostyler/geostyler-style/pull/193).

I think the parsers will need to update to this geostyler-style version before the cli will work smothly again.

**Comment by geraldo** [comment]
Thanks a lot, the release of the new geostyler-style fixed this issue.
